build.sbt

```diff
@@ -3,20 +3,20 @@ organization := "com.algorithmia"
 
 version := "0.9.6-SNAPSHOT"
 
-scalaVersion := "2.11.12"
+scalaVersion := "2.13.0"
```

Comment:
I believe when you do cross-building, you don't need to specify this line?

Reply:
This line determines the "default" scala version when doing things like running `sbt test` locally.

Reply:
If you want to use Jenkins to automate tests across all versions (so each supported Scala version gets tested), here is how we do it in Logpig:

* build.sbt - https://github.com/algorithmiaio/logpig/blob/eb427936461d049012a55e53bd4f9477e32d44d2/build.sbt
* Jenkinsfile - https://github.com/algorithmiaio/logpig/blob/master/Jenkinsfile